designs/dragonphy_top/constraints/gen_constraints.py

```diff
@@ -213,8 +213,6 @@
     iacore/*pi_out_meas* \\
     iacore/*del_out_rep* \\
     iacore/*inbuf_out_meas* \\
-    iacore/*pfd_inp_meas* \\
-    iacore/*pfd_inn_meas* \\
```

Comment:
This was generating warnings, because these pins were removed awhile ago by @sjkim85.  I updated the constraints to reflect this.